app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -1113,6 +1114,10 @@ public static synchronized BgReading bgReadingInsertFromG5(double calculated_val
             bgr.calculated_value = calculated_value;
             bgr.raw_data = SPECIAL_G5_PLACEHOLDER; // placeholder
             bgr.appendSourceInfo("G5 Native");
+            // send both G5 and G6 native for legacy compat
+            if (Ob1G5CollectionService.usingG6()) {
```

Comment:
I don't think it is the source info which is used by legacy AAPS to make decisions about whether to enable SMB, it is the source description. Its only at the point of local intent broadcast that this compatibility needs to be introduced afaik which is why I did it there.